pkg/antctl/raw/traceflow/command_test.go

```diff
@@ -153,3 +154,276 @@ func TestParseFlow(t *testing.T) {
 		}
 	}
 }
+
+func TestRunE(t *testing.T) {
+	tc := struct {
+		name string
+		arg1 *cobra.Command
+		arg2 []string
+	}{
+		name: "test-runE",
+		arg1: Command,
+		arg2: []string{},
+	}
+
+	t.Run(tc.name, func(t *testing.T) {
+		err := runE(tc.arg1, tc.arg2)
+		assert.NoError(t, err)
+	})
+}
+
+func TestOutput(t *testing.T) {
```

Comment:
The reasonable way to test output is to check the output result. I know current function print the result directly to os.stdout via fmt.Print, Can you refactor it so we can catch the output and compare it with expected result?
You may take this as reference https://github.com/antrea-io/antrea/blob/main/pkg/antctl/output/output.go to add a new parameter `writer io.Writer` for different output function. Better to check if possible to reuse those functions on `output.go` directly.